Modules/Bridge/VtkGlue/test/itkImageToVTKImageFilterTest.cxx

```diff
@@ -20,29 +20,84 @@
 
 #include "itkRandomImageSource.h"
 
+#include "vtkMatrix3x3.h"
+
 int itkImageToVTKImageFilterTest(int, char *[])
 {
-  using ImageType = itk::Image<float, 2 >;
+  const int dim = 3;
+  using ImageType = itk::Image<float, dim >;
   using SourceType = itk::RandomImageSource<ImageType>;
+  using SpacingType = SourceType::SpacingType;
+  using OriginType = SourceType::PointType;
+  using DirectionType = SourceType::DirectionType;
   using ConnectorType = itk::ImageToVTKImageFilter<ImageType>;
 
   ImageType::SizeType size;
-  size.Fill(20);
+  size[0] = 40;
+  size[1] = 10;
+  size[2] = 20;
   SourceType::Pointer source = SourceType::New();
   source->SetSize(size);
 
-  ConnectorType::Pointer connector = ConnectorType::New();
-  connector->SetInput(source->GetOutput());
+  SpacingType spacing;
+  spacing[0] = 0.5;
+  spacing[1] = -2;
+  spacing[2] = 1;
+  OriginType origin;
+  origin[0] = -1.5;
+  origin[1] = 0.222;
+  origin[2] = 0;
+  DirectionType direction;
+  direction.Fill(0.0);
+  direction[0][1] = 1;
+  direction[1][0] = -1;
+  direction[2][2] = 0.7;
+  source->SetSpacing(spacing);
+  source->SetOrigin(origin);
+  source->SetDirection(direction);
+
+  source->Update();
+  auto input = source->GetOutput();
+  input->Print(std::cout);
 
+  ConnectorType::Pointer connector = ConnectorType::New();
+  connector->SetInput(input);
   connector->UpdateLargestPossibleRegion();
+  connector->Update();
 
-  connector->GetOutput()->Print(std::cout);
-  connector->GetImporter()->Print(std::cout);
-  connector->GetExporter()->Print(std::cout);
+  // connector->GetImporter()->Print(std::cout);
```

Comment:
I am guessing this is WIP?

Reply:
There were a lot of print calls which were not testing anything and where pretty verbose so I commented them. Didn't remove them because I figured someone could just uncomment them and inspect the log if the tests failed, but happy to either remove them entirely or uncomment them if that's preferred. Same for other test I believe.

Reply:
I don't have a strong opinion on this, but I would prefer for these to either be kept (that might allow easier debugging in the future, and it the test pass we don't have to look at it) or completely removed (in a separate commit that has a short explanation). The second solution would allow one to understand why this was removed when looking at the git log.

Reply:
I'll remove them since they're not useful for the test.